tex/gregoriotex-spaces.tex

```diff
@@ -1457,6 +1473,9 @@
     {spaceporrectuspunctummorashift}{\gre@rubberfalse}%
     {raresignshift}{\gre@rubberfalse}%
     {divisiofinalissep}{\gre@rubberfalse}%
+    {lineskip}{\gre@rubberfalse}%
```

Comment:
The default values for those three led me to believe they were dimensions, but after a bit more research I realize they are skips whose stretch and shrink are 0 by default.  I'll fix that.